dockstore-webservice/src/main/java/io/dockstore/webservice/core/DelimitedValuesConverter.java

```diff
@@ -54,7 +53,7 @@ public String convertToDatabaseColumn(List<String> list) {
 
     @Override
     public List<String> convertToEntityAttribute(String string) {
-        if (StringUtils.isEmpty(string)) {
+        if (string == null) { // The database column may intentionally store an empty string
```

Comment:
Too tired, but not sure if this is the same behaviour on a database column with " "
May be worth unit test just to quickly check, it possible the old version would return a list with no elements. The new version might return a single element list with " "

Reply:
Also don't know if it matters

Reply:
Reverted, no longer applicable